api/sheets/get_property.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Get Node Property
+nav_order: 7
+parent: Google Sheets
+grand_parent: API
+---
+
+# Get the values of a given property for a node.
+
+## =DCPROPERTY(dcids, property)
+
+Given a property DCID, together with a string DCID, a row of cells of DCIDs, or a column of cells of DCIDs, get the values of the given property for those DCIDs.
+
+**Arguments**
+*    `dcids` - DCIDs to get the properties of
+*    `property` - property to get
+
+**Returns**
+
+The properties of the DCIDs. For a single DCID, the result is a column of properties. For a row of DCIDs, the result is a matrix where each column is the members of the DCID at the column's index. For a column of DCIDs, the result is a matrix where each row is the members of the DCID at the row's index.
```

Comment:
I feel here should call out a list of values corresponding to one property. A column of properties sound like multiple properties.

Reply:
Done.